src/librustc_passes/diagnostics.rs

```diff
@@ -310,6 +310,18 @@ loop {
     break;
 }
 ```
+"##,
+
+E0670: r##"
+Rust 2015 does not permit the use of `async fn`.
+
+Example of erroneous code:
+
+```compile_fail,E0670
+async fn foo() {}
+```
+
+You can switch to Rust 2018 edition (or you can remove the keyword `async`).
```

Comment:
```suggestion
Switch to the Rust 2018 edition to use `async fn`.
```
An imperative tone is more in keeping with the error code documentation. Also, I don't think removing the `async` keyword in general is a good suggestion, because if the user has actually used `async` features (i.e. it's not just an empty function), then removing the keyword isn't going to fix anything, because the signature is now incorrect and `await` won't work.